ariadne/asgi.py

```diff
@@ -25,6 +26,7 @@
     DATA_TYPE_MULTIPART,
     PLAYGROUND_HTML,
 )
+from .contrib.subscriptions.subscription import Subscription
```

Comment:
Core features shouldn't rely on contrib package. Contrib is for extra opt-in features. This class can be moved to `ariadne.subscriptions`.

Reply:
Ok